features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/conf/locales/js/i18nResources.json

```diff
@@ -18,7 +18,8 @@
         "charsLeft": "Characters left: ",
 		"replyAdded": "Reply Successfully Added",
 		"callBackHelpMsg": "This parameter can be used in the authorization code grant type when invoking an API. For example, assume that both the client and the API Gateway are run on the same server and the Authorization API URL is https://localhost:8243/authorize. <strong> query component:</strong> “response_type=code&client_id=<consumer_key>&scope=PRODUCTION&redirect_uri=<application_callback_url>” and <strong>headers:</strong> Content-Type:application/x-www-form-urlencoded",
-		"tokenHelpMsg" : "The period after which the token will expire after generation. A negative value ensures that the token never expires."
+		"tokenHelpMsg" : "The period after which the token will expire after generation. A negative value ensures that the token never expires.",
+        "curlHelpMsg" : "CURL command created from the access tokens to send the request."
```

Comment:
Formatting is corrected by now. Please see : https://github.com/wso2/carbon-apimgt/blob/release-1.10.x/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/conf/locales/js/i18nResources.json#L22